### DISCUSSION TITLE
買不到快篩？工程師一天內迅速打造「找找快篩試劑」Chatbot

### DISCUSSION THREAD

**Discussion by Yao900108** [1101-GS4719-C/Python-Community-GS4719-C#580] [科技新知]
還記得 2020 年新冠肺炎剛爆發時，不少民眾見面的第一句話都是「口罩哪裡買」嗎？而現在同樣的情況也發生在快篩試劑上，「你知道哪裡還有快篩試劑嗎？」這句話已成為近期民眾見面的重要對話。
即使開放快篩試劑實名制，到了藥局撲空，或者找不到銷售藥局的狀況仍層出不窮。AI 社群技術專家、微軟最有價值專家（Microsoft Most Valuable Professional）資深程式設計師陳葵懋，以微軟 Azure AI 為核心，一天內快速打造「找找快篩試劑」LINE 聊天機器人，解決民眾的燃眉之急。民眾只要輸入位置，「找找快篩試劑」可立即告知附近藥局與快篩劑庫存，用人工智慧技術解決民眾最關心的問題。

https://technews.tw/2022/05/04/line-chat-bot/

**Comment by PoooYen** [comment]
這個工程師之前就有推出口罩地圖之類的產品，台灣有他真好

**Comment by yenwei12138** [comment]
就算預測到了，太晚去排隊還是會買不到的感覺，畢竟台灣人很愛排隊呢

**Comment by HuanYuuuuuu** [comment]
![image](https://user-images.githubusercontent.com/99785986/166629021-5a84ef01-7f1b-44cc-98d4-0adee5ca8504.png)

這個line機器人設定的好可愛

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，有這些人真好。

**Comment by cordydeng** [comment]
這個應用感覺非常的有幫助，尤其是在現在疫情滿天飛的狀況，但快篩的數量不夠也是既定事實，根本問題還是要取得足夠的快篩

**Comment by HsuChen88** [comment]
疫苗可能也需要有一個統整的地方，不然現在資訊都分散在各家診所的粉專，很難找。

**Comment by Lai27330** [comment]
好貼心的聊天機器人，對於現在嚴重疫情，能找到快篩確定自己的健康，對於各式各樣的工作進行真的非常有幫助。

**Comment by Jay-722** [comment]
這個項目感覺真的蠻方便的，不用到處詢問

**Comment by handsomeniko** [comment]
快篩計之前真的難買

**Comment by lx3211** [comment]
跟之前有一個買不到麥當勞冰炫風就做了冰淇淋地圖的美國工程師好像XD

**Comment by jcpan821** [comment]
能有效降低民眾買快篩撲空的機率，真是太棒了

**Comment by Sazmafuji** [comment]
能夠有效率解決此事好棒

**Comment by bloodtrail0511** [comment]
有這些機器人真的會讓生活方便很多

**Comment by 110409538** [comment]
目前也有很多類似的找快篩網站，但和聊天機器人對話蠻有趣的。

**Comment by Rosancl** [comment]
推推，很有幫助 👍 每次看到有人自發性做這些提供給大家就覺得好暖！

**Comment by d26488399** [comment]
太棒了，這樣就不怕買不到了

**Comment by yucheng1028** [comment]
太酷了八

**Comment by minniewei** [comment]
現在真的太多厲害的工程師了，竟然能在一天做出這麼厲害的東東

**Comment by henrywen547** [comment]
對於現在的狀況可能真的需要聊天機器人來幫助人們紓解日常緊繃的心情

**Comment by liangyinglly** [comment]
這個機器人好適合推薦給老一輩的使用 解決他們不太會上網搜集資料的問題

**Comment by stan890425** [comment]
在之前在缺快篩時真的很方便

**Comment by alexthebigpp** [comment]
感覺不只可以用在快篩，之後如果有其他東西改一改又可以馬上用了。

**Comment by bloodtrail0511** [comment]
大家都在搶快篩，只有我室友給他免費的他還不要==

**Comment by Jimmy911218** [comment]
謝謝分享，課已讓人更方便了
